src/api/SearchAndCompareApi.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\domain\SearchAndCompareDomain.csproj" />
+    <ProjectReference Include="..\domain\SearchAndCompareApiClient.csproj" />

```

Comment:
do folder directory need to change?

Reply:
That's in a new card along with namespace change